tasks/install-ubuntu-cuda-repo.yml

```diff
@@ -4,23 +4,26 @@
     repo: ppa:graphics-drivers/ppa
     state: absent
 
-- name: add pin file
-  copy:
-    src: "cuda-ubuntu.pin"
-    dest: "/etc/apt/preferences.d/cuda-repository-pin-600"
-    owner: "root"
-    group: "root"
-    mode: "0644"
```

Comment:
do we need to remove this file for older installations?

Reply:
Hmm - let me test this

Reply:
Looks like it's fine for this file to remain in place